actionpack/lib/action_dispatch/http/request.rb

```diff
@@ -85,6 +85,9 @@ def controller_class
       end
     end
 
+    # Returns true if the request has a header matching the given key parameter.
+    #
+    #    request.key? :ip_spoofing_check # => true
```

Comment:
Just one last thing, can you make `#=>` instead of `# =>`, notice the space that is present as of now, which should be remvoed.